genie-common-internal/src/main/java/com/netflix/genie/common/internal/configs/CommonServicesAutoConfiguration.java

```diff
@@ -90,7 +99,39 @@ public JobArchiveService jobArchiveService(
     public JobDirectoryManifestService jobDirectoryManifestService(
         final DirectoryManifest.Factory directoryManifestFactory
     ) {
-        return jobDirectoryPath -> directoryManifestFactory.getDirectoryManifest(jobDirectoryPath, false);
+        return new JobDirectoryManifestServiceImpl(directoryManifestFactory, false);
+    }
+
+    /**
+     * Provide a cache manager for the caches used by various components.
+     *
+     * @param caches the available cache beans
+     * @return a {@link CacheManager}
+     */
+    @Bean
+    @ConditionalOnMissingBean(CacheManager.class)
+    CacheManager cacheManager(final List<Cache> caches) {
```

Comment:
Probably doesn't matter but I think the rest of the beans are public

Reply:
fixed